cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -755,6 +770,15 @@ func main() {
 		debug = log.New(os.Stderr, "", log.LstdFlags)
 	}
 
+	if *indexingMetricsReposAllowlist != "" {
+		for _, repo := range strings.Split(*indexingMetricsReposAllowlist, ",") {
+			if repo != "" {
+				reposWithSeparateIndexingMetrics[repo] = struct{}{}
+			}
+		}
+	}
+	debug.Printf("capturing separate indexing metrics for: %s", reposWithSeparateIndexingMetrics)
```

Comment:
nit: maybe it is worth just printing the keys? I think the format above will print `capturing ... for: {<key>: {} <key>: {}}` , which seems hard to read if the list has more than 1 entry.

Reply:
adjusted the output in the latest commit